src/Tests/InstrEngineTests/NaglerInstrumentationMethod/dllmain.cpp

```diff
@@ -80,4 +81,20 @@ STDAPI DLLEXPORT(DllGetClassObject, 12)(_In_ REFCLSID rclsid, _In_ REFIID riid,
     }
 
     return E_NOINTERFACE;
+}
+
+void AssertLogFailure(_In_ const WCHAR* wszError, ...)
+{
```

Comment:
This is still necessary because of the dependency on Common.Lib/Macros.h for xml.